src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -240,6 +240,33 @@ install_packages (RpmOstreeTreeComposeContext  *self,
     if (!rpmostree_context_setup (self->corectx, tmprootfs_abspath, NULL, self->treespec,
                                   cancellable, error))
       return FALSE;
+
+#define TMP_SELINUX_ROOTFS "selinux.tmp/etc/selinux"
+
+    /* By default, the core starts with the SELinux policy of the root, but if we have a
+     * previous commit, it's much likelier that its policy will be closer to the final
+     * policy than the host system's policy. And in the case they match, we skip a full
+     * relabeling phase. Let's use that instead. */
```

Comment:
This makes sense, although I think it'd still be worth having some mechanism to mark pkgcache entries as compatible with multiple policies or something down the line.

Reply:
I'm not sure it'd be truly worth it TBH. I think we discussed this somewhere in the context of the upgrader path, but the same applies here. I.e. we rarely go *backwards* to a previous sepolicy, and e.g. the "new compatible policy" path would still require actually trying to relabel all the pkgs to find out if it was compatible or not (by which time, you've already done the work anyway).